tools/generate_static_provisioning_yamls.sh

```diff
@@ -110,7 +110,7 @@ apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
   name: pvc-${volname}
-  namespace: ibm-spectrum-scale-csi
+  namespace: <PVC namespace> 
 spec:
   accessModes:
     - ${accessmode}
```

Comment:
add `volumeName: ${volname}` in pvc. Add it at same level as accesscode and resources

Reply:
Done